neqo-transport/src/connection/mod.rs

```diff
@@ -594,9 +594,25 @@ impl Connection {
     fn make_resumption_token(&mut self) -> ResumptionToken {
         debug_assert_eq!(self.role, Role::Client);
         debug_assert!(self.crypto.has_resumption_token());
+        // Values less than GRANULARITY are ignored when using the token, so use 0 where needed.
```

Comment:
Do I understand correctly, that you are referring to `RttEstimate::set_initial` here?

https://github.com/mozilla/neqo/blob/75372c214c5a5d1d220e5cc502afffad8d820e21/neqo-transport/src/rtt.rs#L64-L70

Reply:
Yes.